xds/build.gradle

```diff
@@ -39,11 +38,16 @@ dependencies {
 
     testCompile project(':grpc-core').sourceSets.test.output
 
-    compileOnly libraries.javax_annotation
+    compileOnly libraries.javax_annotation,
+            // The unshaded netty_epoll should not be a transitive dependency.
```

Comment:
This comment sort of just reexplains what a compileOnly dependency is. Instead how about:
```
      // At runtime use the epoll included in grpc-netty-shaded
```

Reply:
Done.